frontend/src/app/testQueue/AoEForm/SymptomInputs.tsx

```diff
@@ -70,21 +70,25 @@ const SymptomInputs: React.FC<Props> = ({
         )}
       </div>
       {!noSymptoms && (
-        <TextInput
-          type="date"
-          label="Date of symptom onset"
-          name="symptom_onset"
-          value={onsetDate}
-          onChange={(e) => setOnsetDate(e.target.value)}
-          min="2020-02-01"
-          max={new Date().toISOString().split("T")[0]}
-          required={Object.keys(currentSymptoms).some(
-            (key) => currentSymptoms[key]
-          )}
-          errorMessage={symptomOnsetError}
-          validationStatus={symptomOnsetError ? "error" : undefined}
-          inputRef={symptomOnsetRef}
-        />
+        <>
+          <Label htmlFor="meeting-time">Date of symptom onset</Label>
+          <span className="usa-hint">mm/dd/yyyy</span>
+          <DatePicker
+            className="maxw-mobile"
+            id="symptom_onset"
+            name="symptom_onset"
+            defaultValue={onsetDate}
+            minDate="2020-02-01"
+            maxDate={new Date().toISOString().split("T")[0]}
+            // onChange={(e) => setOnsetDate(e.target.value)}
+            required={Object.keys(currentSymptoms).some(
+              (key) => currentSymptoms[key]
+            )}
+            // errorMessage={symptomOnsetError}
+            // validationStatus={symptomOnsetError ? "error" : undefined}
+            // inputRef={symptomOnsetRef}
```

Comment:
This component appears to take a `ref` prop, so this could probably change to:

```javascript
ref={symptomOnsetRef}
```

This ref is used to focus on the input when the field fails validation. So we'll definitely want to make sure the validation messages are displayed somehow

Reply:
Done in 57f8597